tests/device_tests/ethereum/definitions/eth/chain-id/1/README.md

```
@@ -0,0 +1,30 @@
+```sh
+python cli.py generate --dev-sign -o <output_folder>
+```
+
+Run the command above in an external directory, then pick the contract call
+you want to test.
+
+(Or you can ask the last release manager for the latest definitions or the ERC-7730 FW maintainer(s))
+
+The generated files are arranged as:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)